src/plugins/advanced_settings/public/management_app/advanced_settings.tsx

```diff
@@ -178,13 +221,15 @@ export class AdvancedSettingsComponent extends Component<
   }
 
   onQueryChange = ({ query }: { query: Query }) => {
+    this.setUrlQuery(query.text);
     this.setState({
```

Comment:
nit: I'm not sure that you need to execute `setState` from here.  You already added listener on `url change`. 
The same comment valid for `clearQuery -> setState` too.

Reply:
Good point, this is not needed. See 4fd7dfdc73aeae17a2ed370d658b88d6e59f429e